windows/ReactNativeWebView/ReactNativeWebView.vcxproj

```diff
@@ -170,7 +170,6 @@
   </Target>
   <ImportGroup Label="ExtensionTargets">
     <Import Project="$(SolutionDir)packages\Microsoft.Windows.CppWinRT.2.0.200316.3\build\native\Microsoft.Windows.CppWinRT.targets" Condition="Exists('$(SolutionDir)packages\Microsoft.Windows.CppWinRT.2.0.200316.3\build\native\Microsoft.Windows.CppWinRT.targets')" />
-    <Import Project="$(SolutionDir)packages\Microsoft.Web.WebView2.1.0.1020.30\build\native\Microsoft.Web.WebView2.targets" Condition="Exists('$(SolutionDir)packages\Microsoft.Web.WebView2.1.0.1020.30\build\native\Microsoft.Web.WebView2.targets')" />
```

Comment:
aren't these required when not using PackageReference? In other words, apps on 67 and below would still require the packages.config and manual webview2 versions and imports, no?

Reply:
Nope not needed because apps v0.67 and below shouldn't be using WinUI 2.8-prerelease or higher since it isn't supported in RNW (we didn't backport the change needed to pull in webview2). So they will be using WinUI 2.7 and below and won't need the webview dependency. Then the macros we added to the WebView module prevent building the code that relies on WebView2.

Reply:
I'm not following... if I'm on 0.67 and I want to use react-native-webview, without webview2, and I want to target winui 2.8 (not prerelease), how will this work?